marketplace/resource_applet.go

```diff
@@ -60,7 +60,7 @@ func readAppTile(d *schema.ResourceData, meta interface{}) error {
 		d.Set("image_hash", nil)
 	}
 
-	d.Set("name", app.Name)
+	d.Set("name", app.Title)
```

Comment:
Ah nice catch

Reply:
Before there was a json annotation so it was converting it correctly to name, but since we aren't using json annotations anymore (no generics in go 😞 ), we have to make sure the names match so that mapstructure will properly build it.